README.md

```diff
@@ -1 +1,13 @@
-# simon-says
\ No newline at end of file
+# simon-says
+
+## Order
+
+1. Build the html
+2. Make the buttons light up when you click them
+3. Play a hardcoded pattern like ['red', 'green', 'blue']
+4. Handle win and lose for hardcoded pattern and user input
+5. Generate a random pattern
+6. Block user input during pattern play
```

Comment:
What do you mean? Specifically, doing/teaching the project with each numbered component? As opposed to (what alternative?)